### PR TITLE
Adding source files, tutorial and tests

### PR DESCRIPTION
**Pull request by kostromins**
[Adding source files, tutorial and tests for first time]

**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [x] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [ ] with [X] to check the box)*

### CODE REVIEW

File: sh.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Exception: STATUS_STACK_OVERFLOW at rip=7FF8D7F2AAB7
```
Comment (syarra) [inline sh.exe.stackdump]: Is that file really needed? It looks like a dump from another file.

Comment (kostromins) [inline sh.exe.stackdump]: You are right, it is not. How can I remove it?

It seems that checks failed, however according to console output my tests passed. What should I do in this case?

Comment (laurentheirendt) [inline sh.exe.stackdump]: Your tests are passing. However, the `sh.exe.stackdump` file is still present. May you delete it and then run `contribute`, option `[3]`? Thanks @kostromins.

Comment (kostromins) [inline sh.exe.stackdump]: I have tried to delete it twice. Fist time with regular delete, second time via TortoiseGit --> Delete. sh.exe.stackdump is created every time I run >>contribute in MATLAB. First time contribute (3) detected modifications (since regular delete removes file only locally and file is created again every time I suppose it is valid scenario to detect modifications not deletion), second time contribute (3) detected deletion and prompted  to accept deletion ( I entere "Y" in order to delete file) and after that prompted for adding back the same file ( I enter "N" in order to not add sh.exe.dumpfile), however it is back again in git. The last option I can think off right now is to add it to .gitignore file. Any suggestions?
Regards

Comment (syarra) [inline sh.exe.stackdump]: Hum, that's a rather odd situation!
We will try to replicate this issue and see how we can fix this in the devTools.

In the meanwhile, I deleted it and updated the `.gitignore`. This file will not bother us again!

**Comment by kostromins** [comment]
Should I do something regarding codecov/project error? I changed nothing regarding this in forked folder, so the error is unexpected.

**Comment by syarra** [comment]
@kostromins, don't worry about this error. It frequently happens in PRs.
It's all good. Thanks for your contribution! 🎉

**Comment by kostromins** [comment]
Great! You are welcome!
Now I suppose it will be merged with master branch and will appear in master branch soon?

**Comment by syarra** [comment]
I will merge it in a few minutes. It will be first on the `develop` branch and later this weekend (or beginning of next week) it will be on the `master` branch.

**Comment by kostromins** [comment]
OK. Thank you.

**Comment by kostromins** [comment]
Hi!
I tried to add tutorial_paint4net.m and commit, however I got the following error:

   -> Please select what you want to do (enter the number): 3
dev>  Your Github username is: kostromins.  (Success) 
dev>  Your Github email is: andrejs.kostromins@gmail.com.  (Success) 
dev>   -> Please define the local path to your fork
       current: C:\Users\User\MATLAB.devTools
       Enter the path (press ENTER to use the current path): C:\Users\User\Fork
dev>   -- Configuration -------- 
dev>      GitHub username:       kostromins
dev>      Local directory :      C:\\Users\\User\\Fork\\fork-cobratoolbox
dev>      Remote fork URL:       https://github.com/kostromins/cobratoolbox.git
dev>      Remote repository URL: https://github.com/opencobra/cobratoolbox.git

      Available features are:
      - feature-paint4net

   -> Please enter the name of the feature (branch) that you want to publish (example: feature-paint4net): feature_paint4net
error: Your local changes to the following files would be overwritten by checkout:
	sh.exe.stackdump
Please commit your changes or stash them before you switch branches.
Aborting
dev>  The <develop> feature (branch) cannot be checked outdev>   -> Your <feature_paint4net> feature (branch) exists locally, but not remotely. Do you want to have your local <feature_paint4net> published? Y/N [Y]:N
error: Your local changes to the following files would be overwritten by checkout:
	sh.exe.stackdump
Please commit your changes or stash them before you switch branches.
Aborting
dev>  The <develop> feature (branch) cannot be checked outdev>   -> Your <feature_paint4net> feature (branch) exists locally, but not remotely. Do you want to have your local <feature_paint4net> published? Y/N [Y]:N
dev>   -> You modified sh.exe.stackdump. Do you want to commit the changes? Y/N [N]: N
dev>   -> Do you want to add the new file tutorials/paint4Net/tutorial_paint4net.m? Y/N [N]: Y
dev>  You have selected 1 files to be added in one commit.
dev>   -> Please enter a commit message (example: "Fixing bug with input arguments"): tutorial_paint4net.m added
dev>  Your commit message has been set. (Success) 
dev>  Pushing 1 change(s) to your feature (branch) <feature_paint4net>
error: src refspec feature_paint4net does not match any.
error: failed to push some refs to 'git@github.com:kostromins/cobratoolbox.git'
Error using submitContribution (line 189)
dev>   [submitContribution] Something went wrong when pushing. Please try again. (Error)

Error in contribute (line 119)
                    submitContribution(reply);

By the way sh.exe.dump is still bothering. How should I pull changes via >>contribute to update local .gitignore.
